Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 
 gem 'benchmark-ips', '>= 2.0'
-
+gem 'activesupport'
```

Comment:
I think it's fine to lock down to the latest stable as of now, and add `:require => false`.